chart/kubeapps/templates/_helpers.tpl

```diff
@@ -138,7 +138,7 @@ Trailing / is required for the react app to work
 */}}
 {{- define "kubeapps.ingressRewrites" -}}
 {{ range .Values.ingress.hosts -}}
-{{- if ne .path "/" -}}
+{{- if (.path) and ne .path "/" -}}
```

Comment:
Looks like this condition is wrong and is producing the rewrite rule even when `path = "/"`, which ends up causing a redirect loop in Kubeapps. Just checked and the correct way to do this is:

```
{{- if .path -}}
{{- if ne .path "/" -}}
      rewrite ^({{ .path }})$ $1/ permanent;
{{- end -}}
{{- end -}}
```

However with #910, we may not need to fix this condition.